### PR TITLE
Dynamic case list icon size

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?169125

Can't be merged until https://github.com/dimagi/commcare-odk/pull/314 is

@TylerSheffels

### CODE REVIEW

File: corehq/apps/app_manager/detail_screen.py [hunk 1]
```diff
@@ -321,8 +322,29 @@ def variables(self):
...
+                for path in item.value.values():
+                    map_item = self.app.multimedia_map[path]
+                    if map_item is not None:
+                        image = CommCareMultimedia.get(map_item.multimedia_id)
+                        if image is not None:
+                            for media in image.aux_media:
+                                width = max(width, media.media_meta['size']['width'])
+        if width == 0:
+            return '13%'
+        return str(width)
```
Comment (TylerSheffels) [inline corehq/apps/app_manager/detail_screen.py]: Don't know if this matters, but is media.media_meta stored as 'XX%'?

Comment (orangejenny) [inline corehq/apps/app_manager/detail_screen.py]: no, media_meta has the pixel dimensions. mobile accepts "X%" or "X" and assumes that the latter is a number of pixels.

**Comment by TylerSheffels** [comment]
Good by me then

**Comment by orangejenny** [comment]
@TylerSheffels mobile PRs are merged now, you want to do the honors for this one?

**Pull request merged**